manifests/klusterletkube111/klusterlet-registration-operator-registration-clusterrolebinding.yaml

```diff
@@ -0,0 +1,13 @@
+# Create the cluster rolebinding so klusterlet can deploy registration agent
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: open-cluster-management:registration-operator:{{ .KlusterletName }}-registration
```

Comment:
can you add a `-111` suffix or some such?

Reply:
+1

Reply:
fixed